api/context.go

```diff
@@ -79,7 +79,7 @@ func (c *Context) WriteCacheable(w http.ResponseWriter, cacheable utils.Cacheabl
 
 	// Write error or response
 	if err != nil {
-		c.sc.Log.Warn("server error %v", err)
+		c.sc.Log.Warn(fmt.Sprintf("server error %v", err))
```

Comment:
We should not be `fmt.Sprintf`ing into the logs. Please use the log structuring:
```suggestion
		c.sc.Log.Warn("server error", zap.Error(err))
```

Reply:
fmt.Sprintf is the equivalent of what was happening pre-introduction of zap right?  Aside from structured logs, what's the real benefit?

I held off specifically on updating to avoid any downstream consequences where we may be parsing logs (datadog metrics, alarms, etc).  Plus there were a few places where I was really sure about how to label specific parameters we're logging and didn't want to spend the time gathering context